__tests__/api.test.js

```diff
@@ -86,3 +89,53 @@ test("if a completely invalid ID is passed will throw a different error 400 Inva
       expect(body.msg).toBe("Invalid request");
     });
 });
+
+describe("challenge 5, get/api/articles", () => {
+  test(`does the article list length match the amount of articles in the database`, () => {
+    return request(app)
+      .get(`/api/articles`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articleList.length).toEqual(13);
+      });
+  });
+
+  test(`does the returning array of objects have matching keys author
+  title
+  article_id
+  topic
+  created_at
+  votes
+  article_img_url
+  comment_count, which is the total count of all the comments with this article_id.`, () => {
+    return request(app)
+      .get(`/api/articles`)
+      .expect(200)
+      .then(({ body }) => {
+        body.articleList.forEach((article) => {
```

Comment:
I would also add an expect statement here to check that the body property in each object has been removed - as we're using the toMatchObject() matcher, this will ignore any additional properties on the object so if there is a body property the tests will still pass